### PR TITLE
CFY-5453 Set `updated_at` field in deployment as a part of deployment update

### PR DESCRIPTION
**Pull request by AviaE**

### CODE REVIEW

File: rest-service/manager_rest/deployment_update/handlers.py [hunk 1]
```diff
@@ -769,11 +770,16 @@ def finalize(self, dep_update):
 
         self.sm.update_deployment(self._deployment_template(dep_update))
 
+        # set the `updated_at` field of the deployment related to the
+        # deployment update
+        deployment = dep_update.deployment_update_deployment
+        deployment['updated_at'] = str(datetime.now())
```
Comment (ran-z) [inline rest-service/manager_rest/deployment_update/handlers.py]: the way we set the date has been changed v. recently; go over this with @nirbir

File: tests/workflow_tests/test_deployment_update.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 import shutil
 import time
 import tempfile
-
+import datetime
 from nose.tools import nottest
```
Comment (ran-z) [inline tests/workflow_tests/test_deployment_update.py]: nose is third party

File: tests/workflow_tests/test_deployment_update.py [hunk 3]
```diff
@@ -2176,3 +2176,28 @@ def test_remove_deployment(self):
         self.assertEquals(len(deployment_update_list), len(undel_depups_ids))
         for i in deployment_update_list.items:
             self.assertIn(i['id'], undel_depups_ids)
+
+    def test_set_updated_at_field_in_deployment(self):
```
Comment (ran-z) [inline tests/workflow_tests/test_deployment_update.py]: why cant this be a unit test?

File: rest-service/manager_rest/file_storage_manager.py [hunk 4]
```diff
@@ -547,8 +548,14 @@ def update_deployment(self, deployment):
         updated_deployment = data[DEPLOYMENTS][deployment_id]
         if deployment.scaling_groups is not None:
             updated_deployment.scaling_groups = deployment.scaling_groups
+        if deployment.updated_at is not None:
+            updated_deployment.updated_at = deployment.updated_at
         self._dump_data(data)
 
+    @staticmethod
+    def _storage_node_id(deployment_id, node_id):
```
Comment (ran-z) [inline rest-service/manager_rest/file_storage_manager.py]: if you took the time to add this, might as well make other node-related methods use it :)

**Comment by AviaE** [comment]
reviewed by @ran-z

**Pull request merged**